website/content/en/docs/kubebuilder/quickstart.md

```diff
@@ -285,8 +285,12 @@ $ make run ENABLE_WEBHOOKS=false
 
 #### Build and push the image
 
-Build the operator image and push it to a registry.
+Before building the operator image, ensure the generated Dockerfile references
+the base image you want. You can change the default "runner" image `gcr.io/distroless/static:nonroot`
+by replacing its tag with another, for example `alpine:latest`, and removing
+the `USER: nonroot:nonroot` directive.
```

Comment:
```suggestion
the `USER: nonroot:nonroot` directive for the entrypoint as the following example. 
```

We need to suggest the users use the endpoint implementation. Also, I do not think that the default Kube image will work properly in OCP. Note that we give permissions which are recommended for the container work properly in OCP and allow it to be executed with random users. See: https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/build/bin/user_setup#L5

It may also something that we could customize via tooling as well.

Reply:
See https://github.com/operator-framework/operator-sdk/pull/2966/files#r419869392.

Reply:
Actually, the user setup and entrypoint stuff is probably no longer required now that we removed the stuff that modifies /etc/passwd.

We should let that disappear as part of the kubebuilder integration/migration.